### PR TITLE
Feat/implement rest of poly effects

### PR DESCRIPTION
**Pull request by Blu-J**

### CODE REVIEW

**Comment by Blu-J** [comment]
image
image


image
image

File: container-runtime/src/Adapters/Systems/SystemForEmbassy/MainLoop.ts [hunk 1]
```diff
@@ -74,6 +63,7 @@ export class MainLoop {
       daemon,
       wait: daemon.wait().finally(() => {
         this.clean()
+        console.error("BLUJ It ways it is done?")
```
Comment (dr-bonez) [inline container-runtime/src/Adapters/Systems/SystemForEmbassy/MainLoop.ts:66]: Probably should remove this

File: core/startos/src/service/config.rs [hunk 2]
```diff
@@ -10,10 +10,27 @@ impl Service {
         ConfigureContext { timeout, config }: ConfigureContext,
     ) -> Result<(), Error> {
         let container = &self.seed.persistent_container;
+        let package_id = &self.seed.id;
+
         container
-            .execute::<Value>(ProcedureName::SetConfig, to_value(&config)?, timeout)
+            .execute::<SetResult>(ProcedureName::SetConfig, to_value(&config)?, timeout)
```
Comment (dr-bonez) [inline core/startos/src/service/config.rs:16]: Is this type still correct for the new api?

**Review by dr-bonez — CHANGES_REQUESTED** [review]

File: core/startos/src/service/config.rs (hunk 2 above)
Comment (Blu-J) [reply core/startos/src/service/config.rs:16]: Not sure, but it is at the moment

**Review by dr-bonez — APPROVED** [review]

File: core/startos/src/service/config.rs (hunk 2 above)
Comment (Blu-J) [reply core/startos/src/service/config.rs]: So, going back to value to say that I don't care

**Pull request merged**